modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/parser/java/JavaExtensionModelParser.java

```diff
@@ -319,16 +357,25 @@ public LicenseModelProperty getLicenseModelProperty() {
     return importedTypes;
   }
 
+  @Override
+  public List<NotificationModel> getNotificationModels() {
+    return notificationModels;
+  }
+
+  public StereotypeModelLoaderDelegate getStereotypeLoaderDelegate() {
+    return stereotypeLoaderDelegate;
+  }
+
   private Optional<XmlDslConfiguration> parseXmlDslConfiguration() {
-    return getInfoFromExtension(
-                                extensionElement,
-                                Xml.class,
-                                org.mule.sdk.api.annotation.dsl.xml.Xml.class,
-                                xml -> new XmlDslConfiguration(
-                                                               xml.getStringValue(Xml::prefix),
-                                                               xml.getStringValue(Xml::namespace)),
-                                xml -> new XmlDslConfiguration(
-                                                               xml.getStringValue(org.mule.sdk.api.annotation.dsl.xml.Xml::prefix),
-                                                               xml.getStringValue(org.mule.sdk.api.annotation.dsl.xml.Xml::namespace)));
+    return mapReduceSingleAnnotation(
+                                     extensionElement,
+                                     Xml.class,
+                                     org.mule.sdk.api.annotation.dsl.xml.Xml.class,
+                                     xml -> new XmlDslConfiguration(
+                                                                    xml.getStringValue(Xml::prefix),
+                                                                    xml.getStringValue(Xml::namespace)),
+                                     xml -> new XmlDslConfiguration(
+                                                                    xml.getStringValue(org.mule.sdk.api.annotation.dsl.xml.Xml::prefix),
+                                                                    xml.getStringValue(org.mule.sdk.api.annotation.dsl.xml.Xml::namespace)));
   }
 }
```

Comment:
are there any tests/test extensions that verify the expected behaviour using the new SDK api stereotype annotations?

Reply:
yes. If this breaks it affects everything using lazy init, munit, serialization, etc